### PR TITLE
Add find-unused action

### PR DESCRIPTION
**Pull request by cbush**

### CODE REVIEW

File: source/index.txt [hunk 1]
```diff
@@ -1,3 +1,5 @@
+.. Don't commit this
+
```
Comment (mongodben) [inline source/index.txt:2]: assuming this shouldn't be here

File: .github/workflows/find-unused.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  pull_request:
+    paths:
+      - "source/**"
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: cbush/docdoctor/actions/find-unused@find-unused-action
```
Comment (mongodben) [inline .github/workflows/find-unused.yml:13]: where is the source code for this GH action?

Comment (mongodben) [reply .github/workflows/find-unused.yml:13]: follow up question: will the above PR be merged prior to this one?

File: source/index.txt (hunk 1 above)
Comment (cbush) [reply source/index.txt:2]: Correct, but it's required to trigger the workflow. I'll remove it before merge.

**Review by mongodben — COMMENTED** [review]
lgtm

---

out of curiosity, how'd you test this out? i always struggle w testing for GH actions.

**Comment by cbush** [comment]
The trick is to push it to an upstream branch.

**Review by mongodben — APPROVED** [review]

**Pull request merged**